manifests/apache.pp

```diff
@@ -80,6 +80,16 @@
     }
   }
 
+  if $pulp::manage_db {
+    file {'/etc/pulp/streamer.conf':
```

Comment:
Actually, guessing this path is just wrong and you meant it to go in `/etc/httpd/conf.d` based upon the streamer.conf template